### DISCUSSION TITLE
slow load

### DISCUSSION THREAD

**Discussion by NazarAli** [filamentphp/filament#8552] [General]
hii 
i have problem slow when paginate
i take more than 2  second and i see it load huge number of page
same this picture 

![Screenshot 2023-09-17 122159](https://github.com/filamentphp/filament/assets/7056487/05dad091-28c5-4804-b5c9-1d6a0842f735)

**Comment by NazarAli** [comment]
and the issue not in debugbar i tray disable route the same load time still 
![Screenshot 2023-09-17 123908](https://github.com/filamentphp/filament/assets/7056487/0cbceb4c-4a61-400f-b55e-853da0b7a1a2)


and in each paginate request take 500KB is this number good or i have wrong in config or use bad way ?

![jjj](https://github.com/filamentphp/filament/assets/7056487/286d08d8-a39e-4dfb-a75d-6469793e70aa)

**Reply by leandrocfe** [reply]
what about debugbar disabled?

**Reply by NazarAli** [reply]
same load time.
but when disable xdebug it reduce load time to 250 ms
